client/state/reader/follows/test/reducer.js

```diff
@@ -49,7 +49,7 @@ describe( 'reducer', () => {
 				'dailypost.wordpress.com': { is_following: true },
 			} );
 			const state = items( original, {
-				type: READER_FOLLOW,
+				type: READER_RECORD_FOLLOW,
 				payload: { url: 'http://data.blog' },
```

Comment:
it's going to go away entirely once we switch to the redux bound follow button, so i'm not too keen on flipping it.